SpotifyIOS.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There are a lot of unrelated changes in this diff. (When I say "diff", please think "pull request". It's the same thing. I'll try to say "pull request" but please consider them interchangeable.) In the future it would be great to break it up a bit. For example, you could have a pull request where the only thing in it is the addition of the spotify framework, with no UI changes. Then a separate diff where you add the cells, etc.